paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -155,33 +152,32 @@ public class PaparazziPlugin : Plugin<Project> {
         task.nonTransitiveRClassEnabled.set(nonTransitiveRClassEnabled)
         task.targetSdkVersion.set(android.targetSdkVersion())
         task.compileSdkVersion.set(android.compileSdkVersion())
-        task.projectResourceDirs.set(
+
+        val localResourcePaths = localResourceDirs
+          .map { layers -> layers.flatten() }
+          .map { dirs -> dirs.map { projectDirectory.relativize(it.asFile) }.asReversed() }
+
+        task.projectResourceDirs.set(localResourcePaths)
```

Comment:
I will checkout your branch to verify the generated res logics here

Reply:
I've verified that the changed behavior is correct. It seems that Preview has a caching issue when using resources with the same name in different resource directories. This old priority order was inferred by Preview, potentially leading to a generated resource priority issue in version 1.3.4. After these changes, I have tested different combinations and confirmed that the snapshots rendered by Paparazzi match the screens in the Android application.

Reply:
> It seems that Preview has a caching issue when using resources with the same name in different resource directories.

@kevinzheng-ap is this a known issue by the Android Studio team?

Reply:
I will double check that next week